examples/helloworld/greeting/greet.pb.go

```diff
@@ -2,7 +2,7 @@
 // versions:
 // 	protoc-gen-go-plugin v0.1.0
 // 	protoc               v3.21.12
-// source: examples/helloworld/greeting/greet.proto
```

Comment:
can you please use the top-level makefile to generate these? if so it won't drift like this as the source relative position will remain the same

Reply:
done